docs/userGuide/contentAuthoring.md

```diff
@@ -161,6 +161,29 @@ Each variable must have an id and the value can be any MarkBind-compliant code f
 
 In any file in your site, you can include the variable by surrounding the variable's id with double curly braces, e.g. <code>{<span></span>{options}}</code>
 
+Variables can refer to other variables that are declared earlier, including helpful variables provided by MarkBind.
+
+<tip-box>
+<div>
+This is allowed, MarkBind helpful variables can be used.<br>
+
+`<span id="right_hand">`<code>{<span/>{glyphicon_hand_right}}</code>`</span>`

```

Comment:
- Let's do `<span></span>` as it's not a proper [empty element](https://developer.mozilla.org/en-US/docs/Glossary/Empty_element).
- We can keep it in a single `<code>` element (and avoid the space in browser) by escaping `<`:
  `<code>\<span id="right_hand">{<span></span>{glyphicon_hand_right}}\</span></code>`